content/docs/forms.md

```diff
@@ -21,15 +21,13 @@ HTML form elements work a little bit differently from other DOM elements in Reac
 </form>
 ```
 
-This form has the default HTML form behavior of browsing to a new page when the user submits the form. If you want this behavior in React, it just works. But in most cases, it's convenient to have a JavaScript function that handles the submission of the form and has access to the data that the user entered into the form. The standard way to achieve this is with a technique called "controlled components".
+ഈ ഫോം  നു സാധാരണ html  ഫോം ഒരു യൂസർ submit ചെയ്യുമ്പോ പുതിയ പേജ് ലോഡ്  ചെയ്യുന്ന ഡീഫോൾട് സ്വഭാവം ഉണ്ട് .react ഇലും നമുക് വേണമെങ്കിൽ ഈ സ്വഭാവം ഉപയോഗിക്കാം എങ്കിലും ഈ ഫോം സബ്മിഷൻ കൈകാര്യം ചെയ്യാനും യൂസർ എന്റർ ചെയ്ത വിവരങ്ങൾ അക്സസ്സ് ചെയ്യാനും ജാവാസ്ക്രിപ്റ്റ് ഫങ്ക്ഷന് എഴുതുന്നതാകും ഉത്തമം . ഇത് ചെയ്യാൻ സഹായിക്കുന്ന ഒരു ടെക്‌നിക്ക്  ആണ് "controlled  componenets "
```

Comment:
ഒരു സാധാരണ html ഫോം യൂസർ സബ്മിറ്റ് ചെയ്യുമ്പോൾ പുതിയ പേജിലേക്ക് തിരിച്ചുവിടപ്പെടുന്നതായുള്ള അതേ സ്വഭാവം തന്നെയാൺ ഈ ഫോമിനും.   

<s>react ഇലും</s> React ലും

Reply:
റീയാക്റ്റിലും എന്നുതന്നെ ഉപയോഗിക്കുമ്പോൾ‌ വായിക്കാൻ എളുപ്പമുണ്ട്

Reply:
But I would suggest let's stick on to **React** itself for consistency.